A66-otel-stats.md

```diff
@@ -144,12 +144,53 @@ the scale to better fit the data.
 This section describes a `CallTracer` approach to collect the client and server
 per-attempt/call metrics. A CallTracer is a class that is instantiated for every
 call. This class has various methods that are invoked during the lifetime of the
-call. On the client side, the CallTracer knows about multiple attempts on the
+call. On the client-side, the CallTracer knows about multiple attempts on the
 same call, and creates a `CallAttemptTracer` object for each attempt, and the
-`CallAttemptTracer` gets invoked during the lifetime of the attempt.
+`CallAttemptTracer` gets invoked during the lifetime of the attempt. On the
+server-side, we have an equivalent `ServerCallTracer`. (There is no concept of
+an attempt on the server-side.)
 
 The OTel plugin will basically be a way of configuring CallTracer factories on
-gRPC clients and servers.
+gRPC channels and servers.
+
+A CallTracer needs to know the channel's target in the canonical form, and the
+full qualified method name for filling in the attributes needed on the metrics.
+Similarly on the server-side, the `ServerCallTracer` needs to know the method of
+the incoming call. Depending on the implementation details, the method may be
+propagated as part of the initial metadata.
+
+The following call-outs are needed on the `CallTracer` -
+
+*   When the call has been created. This call-out should be before payload
+    serialization.
+*   When new attempts are created on the call along with information on whether
+    the attempt was a transparent retry or not. (Attempts are created after name
+    resolution but before the pick.) This is also when it's expected for the
```

Comment:
Maybe also mention that it's after xDS HTTP filters are executed, and reference gRFC A39.

Reply:
Done